apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/index.js

```diff
@@ -4,28 +4,47 @@
  * External dependencies
  */
 import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
+import './style.scss';
 
 domReady( () => {
 	const { closeButtonLabel, closeButtonUrl, editorPostType } = fullSiteEditing;
 
+	// Don't alter anything on the post editor.
+	if ( 'post' === editorPostType ) {
```

Comment:
I might suggest using a "whitelist" approach rather than a "blacklist" approach. I.e. return early if it's not the page or not the wp_template. This way, if there are other CPTs (Custom Post Types) that use Gutenberg, they would not be affected either.

Reply:
I definitely think your suggestion is the more wise approach.  I have updated it to a "whitelist" approach!